### ISSUE TITLE
Не падать, если нет локаций

### ISSUE THREAD

**Issue by realsonic** [realsonic/ifml2#4]
Сейчас Плеер показывает стектрейс, если нет локаций. Нужно обходить эту ситуацию в движке. Например, позволять что-то делать, не относящееся к локациям. Рассмотреть, насколько это сложно (например, системная переменная ТекущаяЛокация будет пустой). #

**Comment by Madzi** [comment]
А на сколько сложно проверить валидность локаций при загрузке и не запускать невалидную игру?

**Comment by realsonic** [comment]
@Madzi если просто не запускаться - мол, нет локаций - то вообще не сложно. Хотя... нужен, видимо, флаг в движке, чтобы не парсить ввод после этого. Также, как при выигрыше или проигрыше, например.

**Comment by Madzi** [comment]
Да. Хорошая идея. Если не хватает каих-то локаций, то вывести: "Данная история недописана или повреждена" и на этом завершить цикл обработки истории. Тоже самое после концовки. "История окончена"

**Comment by realsonic** [comment]
Вообще, был запрос на завершение игры - [есть на это задача](https://ifml2.myjetbrains.com/youtrack/issue/ifml2-428271). А запрос был от автора "Вокруг веба", т.к. он позволяет поиграть в две ЯРИЛьные игры: [Комсомолец](https://vk.com/wall-75349301_28) и [Лабиринт фавна](https://vk.com/wall-75349301_25). Он меня не дождался и сделал своими тегами в коде игр.
